modules/customize-certificates-api-add-named.adoc

```diff
@@ -12,11 +12,17 @@ load balancer is used.
 
 .Prerequisites
 
-* You must have the certificate and key, in the PEM format, for the
-client's URL.
+* You must have a certificate for the client's URL and its private key.  Each
```

Comment:
Please use single spaces between sentences. I think it renders fine on docs.openshift.com, but I'm not sure how it does on the Customer Portal. Thanks!